### PR TITLE
ASP.NET Core 7 breaking change for MVC empty body detection

### PR DESCRIPTION
**Pull request by serpent5**
Documents https://github.com/aspnet/Announcements/issues/484.

[Internal Review](https://review.docs.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding?branch=pr-en-us-28905).

### CODE REVIEW

File: docs/core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+## Recommended action
+
+No changes are required. However, if you're seeing unexpected behavior, ensure that client requests send the appropriate HTTP headers.
+
+## Affected APIs
+
+MVC controller actions
+
+---
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding.md:92]: Did you add this line `---` on purpose?

Comment (serpent5) [reply docs/core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding.md:92]: Oh no, sorry, I use that to separate the existing article I copy and the new content, but then I'm supposed to remove it at the end. 🤦

File: docs/core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+To align with other parts of the framework that use <xref:Microsoft.AspNetCore.Http.Features.IHttpRequestBodyDetectionFeature.CanHaveBody%2A?displayProperty=nameWithType> and to fix [Optional `[FromBody]` Model Binding is not working (dotnet/aspnetcore #29570)](https://github.com/dotnet/aspnetcore/issues/29570).
+
+## Recommended action
+
+No changes are required. However, if you're seeing unexpected behavior, ensure that client requests send the appropriate HTTP headers.
+
+## Affected APIs
+
+MVC controller actions
+
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding.md:91]:
```suggestion
```

**Review by gewarren — APPROVED** [review]
LGTM - thanks!

**Pull request merged**